staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller.go

```diff
@@ -310,21 +317,21 @@ func (c *celAdmissionController) Validate(
 				versionedAttr = va
 			}
 
-			decisions, err := bindingInfo.validator.Validate(versionedAttr, param)
+			decisions := bindingInfo.validator.Validate(versionedAttr, param)
```

Comment:
I guess here it no longer returns an error but rather just turns it into a policy decision. Should I revert that?

Reply:
I think its handling compilationErrors the same way https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/validator.go#L213-L224 within the evaluation stage

Reply:
This _looks_ right to me when navigating the review on github. I had to cross-reference a few things but it all checked out. @cici37 are there any tests missing that we need to make sure the behavior remains as expected?